### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by sinnec**

### CODE REVIEW

**Comment by sinnec** [comment]
@gcharis @momegas Do you think the auth stuff that use the token should be removed for now?

**Comment by sinnec** [comment]
> > @gcharis @momegas Do you think the auth stuff that use the token should be removed for now?
> 
> What do you mean "the token"? Which calls are you refering to?

I mean the auth endpoint and functions that use a token to authorize the user.

**Comment by momegas** [comment]
Yeah, any code not in use should be removed

**Review by momegas — COMMENTED** [review]
```python
if not authenticated:
        return errors.unauthorized()
```

I think this needs to be in a middleware

File: src/tests/v1/test_drifting_metrics.py [hunk 1]
```diff
@@ -17,7 +20,10 @@ def test_drifting_metric_get_model_multi_class_all(client):
 
 @pytest.mark.order(get_order_number("drifting_metrics_get_model_binary_all"))
 def test_drifting_metric_get_model_binary_all(client):
-    response = client.get(f"/v1/models/{state.model_binary['id']}/drifting_metrics")
+    response = client.get(
+        f"/v1/models/{state.model_binary['id']}/drifting_metrics",
```
Comment (momegas) [inline src/tests/v1/test_drifting_metrics.py:24]: I just noticed this. we shouldn't have underscores in our endpoints. It should be `drifting_metrics`

**Pull request merged**